gitdash/yarn.lock

```diff
@@ -0,0 +1,4792 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Can you also please remove the yarn.lock file?

Reply:
Deleted the yarn.lock file. Sorry about the yarn and npm issue. I will use npm commands in the future.